cerberus-web/src/test/java/com/nike/cerberus/dao/SecureDataVersionDaoTest.java

```diff
@@ -56,7 +57,7 @@
           .setSdboxId(sdbId)
           .setAction(action.name())
           .setPath(path)
-          .setEncryptedBlob(encryptedBlob.getBytes())
+          .setEncryptedBlob(encryptedBlob.getBytes(Charset.forName("UTF-8")))
```

Comment:
Is this needed?

Reply:
It was resulting in find bugs errors when we didn't set the Charset.